source/shared/cpp/ObjectModel/MarkDownBlockParser.cpp

```diff
@@ -177,9 +199,9 @@ void EmphasisParser::UpdateCurrentEmphasisRunState(DelimiterType emphasisType)
     m_currentDelimiterType = emphasisType;
 }
 
-bool EmphasisParser::IsRightEmphasisDelimiter(char ch)
+bool EmphasisParser::IsRightEmphasisDelimiter(int ch)
```

Comment:
>int [](start = 46, length = 3)

char

Reply:
and other places

---
In reply to: [158142538](https://github.com/Microsoft/AdaptiveCards/pull/1053#discussion_r158142538) [](ancestors = 158142538)

Reply:
i need to change it to 'int' to handle eof

---
In reply to: [158142607](https://github.com/Microsoft/AdaptiveCards/pull/1053#discussion_r158142607) [](ancestors = 158142607,158142538)